css-positioning/fixed.css

```diff
@@ -0,0 +1,13 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  top: 100px;
+  left: 300px;
```

Comment:
Try just making this element the full width of the page. It already has `text-align: center;`, which will handle centering it